mvpa2/tests/test_searchlight_hyperalignment.py

```diff
@@ -153,6 +153,8 @@ def test_hyperalignment_measure(self):
         mappers_fsf_same = fsha_fsf_same(dss_rotated)
         mappers_fsn = fsha_fsn(dss_rotated)
         mappers = fsha(dss_rotated_clean)
+        mappers_diffsizedss = fsha_fsf([sd[:, nfs] for nfs, sd in
```

Comment:
I don't see this `mappers_diffsizedss` being checked anywhere below -- so is it just a smoke test which would have led to crash with prev implementation?
NB When I saw "Adds test to break current version on master without this fix" I felt like a dad whose 1st grader brought home his first A ;)